app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from typing import Callable
+
+
+class CommandError(Exception):
+    """Checks command input for right format."""
+
+
+def check_command(func: Callable) -> Callable:
+    def wrapper(command: str) -> Callable:
+        if command.split()[0] != "cp" or len(command.split()) < 3:
+            raise CommandError("Please enter valid command!")
+        return func(command)
+
+    return wrapper
+
+
+@check_command
+def copy_file(command: str) -> None:
+    source_name, target_name = command.split()[1:3]
```

Comment:
Consider using unpacking here without slicing, e.g. 
```suggestion
    _, source_name, target_name = command.split()
```